src/main/scala/scalismo/faces/io/PixelImageIO.scala

```diff
@@ -48,8 +50,23 @@ object PixelImageIO {
     val bufImage = converter.toBufferedImage(image)
     file match {
       case f if f.getAbsolutePath.toLowerCase.endsWith(".png") => javax.imageio.ImageIO.write(bufImage, "png", file)
-      case f if f.getAbsolutePath.toLowerCase.endsWith(".jpg") => javax.imageio.ImageIO.write(bufImage, "jpg", file)
+      case f if f.getAbsolutePath.toLowerCase.endsWith(".jpg") => {
+        writeJPEGToStream(image, new FileOutputStream(file), 1.0)
```

Comment:
It is already converted to a `BufferedImage` so please pass here `bufImage` instead of `image`. The curly braces are not required and could be removed for consistency.